api/signup_test.go

```diff
@@ -153,7 +153,7 @@ func (ts *SignupTestSuite) TestWebhookTriggered() {
 
 	w := httptest.NewRecorder()
 	ts.API.handler.ServeHTTP(w, req)
-	assert.Equal(http.StatusOK, w.Code)
+	// assert.Equal(http.StatusOK, w.Code)
```

Comment:
Is commenting out intended?

Reply:
whoops, nope this wasn't intended, thanks for catching this!